unliftio/src/UnliftIO/Concurrent.hs

```diff
@@ -0,0 +1,180 @@
+{-# LANGUAGE RankNTypes #-}
+-- | Unlifted "Control.Concurrent".
+--
+-- Use this module only if "UnliftIO.Async" is not enough.
```

Comment:
Thank you for including this! Perhaps it's also worth nothing that, unlike most other modules in the package, this one is not reexported by `UnliftIO`.